[2025-07-26T11:43:18.298+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-26T11:43:18.317+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reset_index_and_pull_holdout.reset_and_pull manual__2025-07-26T11:43:17.503964+00:00 [queued]>
[2025-07-26T11:43:18.324+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reset_index_and_pull_holdout.reset_and_pull manual__2025-07-26T11:43:17.503964+00:00 [queued]>
[2025-07-26T11:43:18.324+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-26T11:43:18.332+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): reset_and_pull> on 2025-07-26 11:43:17.503964+00:00
[2025-07-26T11:43:18.338+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=25971) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T11:43:18.339+0000] {standard_task_runner.py:64} INFO - Started process 25972 to run task
[2025-07-26T11:43:18.339+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'reset_index_and_pull_holdout', 'reset_and_pull', 'manual__2025-07-26T11:43:17.503964+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/reset_dag.py', '--cfg-path', '/tmp/tmpuw9iyxoy']
[2025-07-26T11:43:18.341+0000] {standard_task_runner.py:91} INFO - Job 40: Subtask reset_and_pull
[2025-07-26T11:43:18.361+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-26T11:43:18.400+0000] {task_command.py:426} INFO - Running <TaskInstance: reset_index_and_pull_holdout.reset_and_pull manual__2025-07-26T11:43:17.503964+00:00 [running]> on host 2992d32593f0
[2025-07-26T11:43:18.444+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='reset_index_and_pull_holdout' AIRFLOW_CTX_TASK_ID='reset_and_pull' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T11:43:17.503964+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-26T11:43:17.503964+00:00'
[2025-07-26T11:43:18.444+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-26T11:43:19.454+0000] {logging_mixin.py:188} INFO - ✅ Đã xóa toàn bộ dữ liệu trong bảng new_reviews!
[2025-07-26T11:43:19.461+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-07-26T11:43:19.465+0000] {logging_mixin.py:188} INFO - current_index reset to 0
[2025-07-26T11:43:19.468+0000] {logging_mixin.py:188} INFO - Will pull s3://recsys-ops/holdout.parquet -> /opt/***/data/holdout.parquet
[2025-07-26T11:43:20.049+0000] {logging_mixin.py:188} INFO - ✅ Đã tải xong holdout.parquet từ S3!
[2025-07-26T11:43:20.051+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-26T11:43:20.052+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-26T11:43:20.068+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=reset_index_and_pull_holdout, task_id=reset_and_pull, run_id=manual__2025-07-26T11:43:17.503964+00:00, execution_date=20250726T114317, start_date=20250726T114318, end_date=20250726T114320
[2025-07-26T11:43:20.117+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-07-26T11:43:20.142+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-26T11:43:20.144+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
